src/styles/announcements.ts

```diff
@@ -0,0 +1,106 @@
+import { createStyles } from "@material-ui/core";
+import theme from "./theme";
+import hero from "../resources/img/mantyharju-images/mantyharju-images/hero-image-mantyharju.jpg";
+
+const { breakpoints } = theme;
+
+const sidePaddingSm = "0 2rem";
+const sidePaddingMd = "0 4rem";
+const sidePaddingLg = "0 6rem";
+const sidePaddingXl = "0 8rem";
+
+export default createStyles({
+
+  heroImageDiv: {
+    display: "flex",
+    alignItems: "center",
+    backgroundImage: `url(${hero})`,
+    backgroundRepeat: "no-repeat",
+    backgroundPosition: "center center",
+    backgroundSize: "cover",
+    color: "#fff",
+    height: "40vh",
+    padding: sidePaddingSm,
+    width: "100%",
+    position: "relative",
+    overflow: "hidden",
+    [breakpoints.up("sm")]: {
+      padding: sidePaddingMd,
+      height: "40vh",
+      fontSize: "1.25em",
+    },
+    [breakpoints.up("md")]: {
+      padding: sidePaddingLg,
+      paddingTop: "6rem",
+    },
+    [breakpoints.up("lg")]: {
+      padding: sidePaddingXl,
+      height: "40vh",
+    },
+    [breakpoints.up("xl")]: {
+      height: "40vh",
+    },
+    "&:after": {
+      zIndex: 1,
+      content: "''",
+      position: "absolute",
+      left: 0,
+      top: 0,
+      width: "100%",
+      height: "100%",
+      backgroundColor: "rgba(0,0,0,0.2)"
+    }
+  },
+
+  heroContent: {
+    position: "relative",
+    zIndex: 2,
+    textShadow: "0 0 25px rgba(0,0,0,0.2)",
+    "& h2": {
+      fontSize: 20,
+      [breakpoints.up("sm")]: {
+        fontSize: 30,
+      },
+      [breakpoints.up("md")]: {
+        fontSize: 35,
+      },
+      [breakpoints.up("lg")]: {
+        fontSize: 40,
+      }
+    }
+  },
+
+  heroText: {
+    fontSize: 30,
+    marginBottom: "2rem",
+    fontWeight: 800,
+    [breakpoints.up("sm")]: {
+      fontSize: 40,
+    },
+    [breakpoints.up("md")]: {
+      fontSize: 45,
+    },
+    [breakpoints.up("lg")]: {
+      fontSize: 45,
+    }
+  },
+
+  cards: {
+    display: "grid",
+    gridTemplateColumns: "repeat(3, 1fr)",
+    margin: "0vw 8vw 4vw 12vw",
```

Comment:
I would not recommend using viewport width (vw) as a unit for margins or paddings. Please use "rem", or theme.spacin( multiplier ), percentages or fixed pixel sizes. First set the margins needed for small mobile devices and then use breakpoints.up("sm") and go up from there to make sure the margin is reasonable in every screensize.